commit 43156d3268f5e1f3f45f7a48c7a1104d94db3da2 (refs/changes/53/1853/3)
Author: Brittany Wald <brittany.wald@joyent.com>
Date:   2017-04-28T22:53:04+00:00 (2 years, 5 months ago)
    
    MANTA-2904 502 responses should use JSON, HTTP 1.1
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/etc/haproxy.cfg.in b/etc/haproxy.cfg.in
index 0dd5a07..e1a4212 100644
--- a/etc/haproxy.cfg.in
+++ b/etc/haproxy.cfg.in
@@ -26,6 +26,7 @@ defaults
         errorfile 400 /opt/smartdc/muppet/etc/400.http
         errorfile 408 /opt/smartdc/muppet/etc/408.http
         errorfile 500 /opt/smartdc/muppet/etc/503.http
+        errorfile 502 /opt/smartdc/muppet/etc/503.http
         errorfile 503 /opt/smartdc/muppet/etc/503.http
         errorfile 504 /opt/smartdc/muppet/etc/503.http
 
